### ISSUE TITLE
use "language" instead of "cyrilliqueName"

### ISSUE THREAD

**Issue by VladimirAlexiev** [dbpedia/mappings-tracker#15]
@jplu: http://mappings.dbpedia.org/index.php/Mapping_fr:Infobox_Ville_de_Serbie would make this:

```
     <belgrade> foaf:name "Belgrade";
        dbo:cyrilliqueName "Белград".
```

But both names have the same stature, and nobody in the world would think of using your made-up property. Instead, you should make this:

```
     <belgrade> foaf:name "Belgrade"@sr-Latn;
       foaf:name "Белград"@sr-Cyrl.
```

I.e. the same prop, and a lang tag to indicate the language.
The field "language" of http://mappings.dbpedia.org/index.php/Template:PropertyMapping is used for this. (The target property must have range rdf:langString, which is true: http://mappings.dbpedia.org/index.php/OntologyProperty:Foaf:name)

Please edit the template, then delete http://mappings.dbpedia.org/index.php/OntologyProperty:CyrilliqueName

**Comment by VladimirAlexiev** [comment]
Same for http://mappings.dbpedia.org/index.php/OntologyProperty:ArabicName

**Comment by jplu** [comment]
Done for both ! Tell me if you see some that I missed.

**Comment by VladimirAlexiev** [comment]
Great! 
- You also need to delete the redirect page that mediawiki makes: http://mappings.dbpedia.org/index.php/Mapping_fr:Infobox_Ville?redirect=no
- Is nom actually in fr?
- Unfortunately the testing doesn't work: http://mappings.dbpedia.org/server/mappings/fr/extractionSamples/Mapping_fr:Infobox_Ville_de_Serbie. The reason is https://github.com/dbpedia/extraction-framework/issues/289. Let's gently bug @jimkont and Volha Bryl to fix it :-)
- But you can still test per-resource, eg: http://mappings.dbpedia.org/server/extraction/fr/extract?title=Belgrade&revid=&format=turtle-triples&extractors=custom. Unfortunately this doesn't have a cyrillic name, could you kindly:
  *\* find a resource that does have a cyrillic name, test it
  *\* add the test URL to the Discussion page of the infobox mapping. We always want test cases for the tricky parts
  *\* Add a section == Testing Best Practices == right after http://mappings.dbpedia.org/index.php/Main_Page#Editorial_Process describing this, and givе a couple test URLs. Here's a BG one: http://mappings.dbpedia.org/server/extraction/bg/extract?title=Лили+Иванова&revid=&format=turtle-triples&extractors=custom.
- Other "<language>name" fields: I'm afraid there are many: http://mappings.dbpedia.org/index.php/What%27s_in_a_Name#Language-specific_Names. 

Thanks for being so cooperative! Sorry if I've been a bit cross... But I think we're now forging editorial rules that will improve dbpedia overall!
Can you come to DBpedia Dublin 2015 which is on 5 Feb?

**Comment by jplu** [comment]
> Is nom actually in fr?

I added the language tag to the mapping.

I created the discussion page http://mappings.dbpedia.org/index.php/Mapping_fr_talk:Infobox_Ville_de_Serbie. Please tell me it it's ok. I found also an example http://mappings.dbpedia.org/server/extraction/fr/extract?title=Po%C5%BEega_(Serbie)&revid=&format=turtle-triples&extractors=custom but apparently it is not working, any idea why ?

> Other "name" fields: I'm afraid there are many:

I can take care of this once I will have a bit more time.

> Thanks for being so cooperative! Sorry if I've been a bit cross... But I think we're now forging editorial rules that will improve dbpedia overall!

No don't worry, it's for something better so it's a pleasure for me to help in anything.

> Can you come to DBpedia Dublin 2015 which is on 5 Feb?

I would like but I have to ask to my PhD supervisor first.

**Comment by VladimirAlexiev** [comment]
> nom@fr

Yes, I saw the tag. But my question is whether in fr.wikipedia, it is actually in French and not eg in sr-Latn (I think it is)

>  not working, any idea why ?

Bug in the extractor: https://github.com/dbpedia/extraction-framework/issues/303

> discussion page

Thanks for the example! Used the example to post the bug report 303.  I reworded it to be simpler/direct, and created this section on the Main Page: http://mappings.dbpedia.org/index.php/Main_Page#Testing_Best_Practices

**Comment by jplu** [comment]
> Yes, I saw the tag. But my question is whether in fr.wikipedia, it is actually in French and not eg in sr-Latn 

If we reffer to the model definition, the property "nom" is the french name of what you are describing. This is why they add other property to define it in differents languages.

> I reworded it to be simpler/direct, and created this section on the Main Page

Great !!

About the DBpedia meeting in Dublin, is-it not the 9th instead of 5th ? Anyway I can certainly come.

**Comment by VladimirAlexiev** [comment]
Yes, 9th Feb

**Comment by jimregan** [comment]
There were also:
germanName
englishName
sardinianName
alemmanicName
ladinName

I've replaced the properties with language tags; should I just go ahead and delete these?

**Comment by jimregan** [comment]
Also:
irishName
manxName
welshName
scotishName (sic)
scottishName
scotsName
cornishName
gaelicName
arberishtName
touaregName (sic)
mozabiteName
kabyleName
chaouiName
tamazightName
dutchName
frioulanName
finnishName
frenchName

**Comment by VladimirAlexiev** [comment]
Thanks @jimregan !!!! 
Would you mind updating this section http://mappings.dbpedia.org/index.php/What%27s_in_a_Name#Language-specific_Names: to say "fixed" in the last col.
And if you know some props that shouldn't be nuked (eg algerianSettlementName, frenchNickname?) move them to their own sec.

**Comment by jimregan** [comment]
Sure, no problem. 'algerianSettlementName' is unused, I think; 'algerianName' I think does not map cleanly to a single language (the examples I saw had both (some dialect of) Berber and Arabic), and 'calabrianName' can't be replaced by a language tag because there isn't one.
What is the "request for deletion" policy for language name properties? The ones I've listed can be safely deleted, but I don't want to delete them without getting at least a +1.

**Comment by jimregan** [comment]
I forgot:
italianName
luxembourgishName
sicilianName
occitanName

Today's victims :)
japanName
maoriName
greekName

**Comment by jimregan** [comment]
The following I would consider different, but note that all are unused:
algerianSettlementName
tamazightSettlementName
touaregSettlementName

**Comment by VladimirAlexiev** [comment]
Kill verything unused!

**Comment by jimregan** [comment]
Done!

**Comment by jimkont** [comment]
Late +1

**Comment by VladimirAlexiev** [comment]
Fixed moldavianName to "mo" and "gagaouze" to "gag".
But "gagaouze" was missing from http://mappings.dbpedia.org/index.php/What%27s_in_a_Name#Language-specific_Names since it doesn't have the word "name". I found it only by coincidence, since I know both of these languages.

@jplu, please help us out here. You know better all the per-language name props you made.

**Comment by VladimirAlexiev** [comment]
@jimregan : there's no procedure for deletion, but since we are 2 people discussing here, that's twice more than any other issue on this tracker or the mapping wiki :-) so I think we have consensus.

Replaced 'calabrianName' with custom lang tag x-calabria.
Replaced 'algerianName' with region-based tag qqq-DZ.

**Comment by jplu** [comment]
The three other I have found:

russianName
kanjiName
frenchNickname

And I have modified the page: http://mappings.dbpedia.org/index.php/What%27s_in_a_Name

**Comment by VladimirAlexiev** [comment]
@jplu Thanks!
frenchNickname perhaps should map not to foaf:name but to otherName? (or alias or somethign like that)?
Then please close it

**Comment by jplu** [comment]
I would say `foaf:nick` could be even better.

**Comment by VladimirAlexiev** [comment]
Agree, but is foaf:nick used elsewhere?

**Comment by jimregan** [comment]
http://mappings.dbpedia.org/index.php/Special:WhatLinksHere/OntologyProperty:Foaf:nick
Yes :)
